torch/utils/backcompat/broadcast/warning/__init__.py

```diff
@@ -0,0 +1,18 @@
+import sys
+from torch._C import _set_backcompat_broadcast_warn
+from torch._C import _get_backcompat_broadcast_warn
+
+
+class Warning:
+    def __init__(self):
+        pass
+
+    def set_enabled(self, value):
+        _set_backcompat_broadcast_warn(value)
+
+    def get_enabled(self):
+        return _get_backcompat_broadcast_warn()
+
+    enabled = property(get_enabled, set_enabled)
+
+sys.modules[__name__] = Warning()
```

Comment:
On 2. I think I did this because properties don't work outside of objects, see https://stackoverflow.com/questions/17007886/property-method-without-class.  At least some quick attempts don't seem to work.